.storybook/foundations/drop-shadow/drop-shadow.stories.js

```diff
@@ -0,0 +1,55 @@
+import { html } from "lit";
+import { classMap } from "lit/directives/class-map.js";
+import "./index.css";
+
+export default {
+  title: "Foundations/Drop shadow",
+  description: "Drop shadows draw attention and give the appearance of depth. By default, this style is used to show elevation, when content appears on top of other content.",
+  component: "Drop shadow",
+  tags: ['foundation'],
+}
+
+const DropShadowSwatch = ({rootClass = "spectrum-Foundations-Example-DropShadow-swatch", variant}) => {
+  return html`
+  <div class=${classMap({
+    [rootClass]: true,
+    [`${rootClass}--${variant}`]: typeof variant !== undefined,
+  })}></div>
+`;}
+
+const DropShadowBackground = ({rootClass = "spectrum-Foundations-Example-swatch-container", color, ...args}) => {
+
+  return html`
+    <div class=${classMap({
+      [rootClass]: true,
+      "spectrum--light": color === "light",
+      "spectrum--dark": color === "dark",
+    })}>
+      ${DropShadowSwatch(args)}
+    </div>
+  `;
+}
+
+const DropShadowVariant = ({...args}) => {
+  return html`
+  <div class="spectrum-Foundations-Example-variant-container">
+    ${DropShadowBackground({...args, color: "light"})}
+    ${DropShadowBackground({...args, color: "dark"})}
+  </div>
+`;}
+
+
+export const DropShadowEmphasizedDefault = DropShadowVariant.bind({});
+DropShadowEmphasizedDefault.args = {
+  variant: "emphasized-default",
+};
```

Comment:
```suggestion
export const DropShadowEmphasizedDefault = DropShadowVariant.bind({});
DropShadowEmphasizedDefault.args = {
  variant: "emphasized-default",
};
DropShadowEmphasizedDefault.parameters = {
	chromatic: { disableSnapshot: true },
};
```
Do we want to exclude these stories from Chromatic? If so, we might want to add this parameter to them all.

Reply:
Is there a reason to disable the snapshots? We certainly could, on the argument that the drop shadow tokens are tested in individual components, but we didn't do this for the other foundations stories (corner-rounding or down state) as far as I can tell.